[bug] rokcetmq env

## env prepare

1. use docker image with commitId `92df95ceea634e764aa88fbfcdb5f1475a01259a` will throw exception


## expectation result

1. cannot send message to broker
```log
Caused by: org.apache.rocketmq.remoting.exception.RemotingTooMuchRequestException: sendDefaultImpl call timeout。
```

## cause reason

1. this is because it uses `Intranet IP`: 192.168.0.3, so it cannot send message to broker.

## Build

1. 2020-07-28

--- a/env/rocketmq/broker.conf
+++ b/env/rocketmq/broker.conf
@@ -28,7 +28,8 @@ brokerId=0
 
 # 启动IP,如果 docker 报 com.alibaba.rocketmq.remoting.exception.RemotingConnectException: connect to <192.168.0.120:10909> failed
 # 解决方式1 加上一句 producer.setVipChannelEnabled(false);，解决方式2 brokerIP1 设置宿主机IP，不要使用docker 内部IP
-# brokerIP1=192.168.0.253
+#  java code 和 broker 在一台 server 上就可以使用 内网 ip
+brokerIP1=101.132.45.28
 
 # 在发送消息时，自动创建服务器不存在的topic，默认创建的队列数
 defaultTopicQueueNums=4

